crates/router/src/types.rs

```diff
@@ -533,6 +534,7 @@ pub struct PaymentsCancelData {
     pub cancellation_reason: Option<String>,
     pub connector_meta: Option<serde_json::Value>,
     pub browser_info: Option<BrowserInformation>,
+    pub metadata: Option<pii::SecretSerdeValue>,
```

Comment:
Please add comment on from where and what info this metadata holds?

Reply:
Done